extensions/amp-intersection-observer-polyfill/0.1/amp-intersection-observer-polyfill.js

```diff
@@ -27,7 +28,9 @@ const TAG = 'amp-intersection-observer-polyfill';
 function upgrade(win) {
   upgradePolyfill(win, () => {
     installIntersectionObserver();
-    // TODO(dvoytenko): hookup 3p host updater for inabox.
+    if (win != win.top) {
```

Comment:
I have a question about this. Not sure how `win.top` work, does this not cover FIE case. 
Also I know the AMP doc can be served from a full screen iframe. Will that affect the check here?